gnomad/utils/liftover.py

```diff
@@ -13,193 +13,148 @@
 logger.setLevel(logging.INFO)
 
 
-def get_checkpoint_path(gnomad: bool, data_type: str, path: str, is_table: bool) -> str:
-    """
-    Creates a checkpoint path for Table
-
-    :param gnomad: Whether data is gnomAD data
-    :param data_type: Data type (exomes or genomes for gnomAD; not used otherwise)
-    :param path: Path to input Table/MatrixTable (if data is not gnomAD data)
-    :param is_table: Whether data is a Table
-    :return: Output checkpoint path
-    """
+GRCH37_to_GRCH38_CHAIN = "gs://hail-common/references/grch37_to_grch38.over.chain.gz"
+"""
+Path to chain file required to lift data from GRCh37 to GRCh38.
```

Comment:
I actually am not sure about this either, but I've found that having some documentation for constants is helpful

Reply:
That's the format for Sphinx's autodoc.

> For module data members and class attributes, documentation can either be put into a comment with special formatting (using a #: to start the comment instead of just #), or in a docstring after the definition. Comments need to be either on a line of their own before the definition, or immediately after the assignment on the same line. The latter form is restricted to one line only.
> – https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute